Makefile

```diff
@@ -8,13 +8,24 @@ GIT_HOOKS := .git/hooks/applied
 obj-m := fastecho.o
 fastecho-objs := $(SRCS:.c=.o)
 
-all: $(GIT_HOOKS)
-	make -C $(KERNEL_DIR) SUBDIRS=$(BUILD_DIR) KBUILD_VERBOSE=$(VERBOSE) modules
+CFLAGS = -Wall -pthread
```

Comment:
Don't confuse build system. Change it to `CFLAGS_user`, which explicitly emphasize on user programs.